charger/wattpilot/wattpilot.go

```diff
@@ -0,0 +1,495 @@
+package wattpilot
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"crypto/sha512"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/signal"
+	"strconv"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"golang.org/x/crypto/pbkdf2"
+)
+
+const (
+	MAX_RECONNECT_RETRIES = 5
+)
+
+//go:generate ./mapping_gen.sh
```

Comment:
Generator bitte in Go, keine Shellskripte- die kann ich leider nicht warten

Reply:
Ja ist geaendert - nach ein paar Anlaufschwierigkeiten damit...